Description: Remove embedded libraries in Irrlicht source (flagged by lintian)
 Since removing the subdirectories containing libjpeg/libpng/zlib/bzip2/lzma
 would make this patch way too large, this patch should be applied on top of
	rm -rf lib/jpeglib/
	rm -rf lib/zlib/
	rm -rf lib/libpng/
 in debian/rules (e.g. in the clean target).
Forwarded: no
Author: Vincent Cheng <vcheng@debian.org>
Last-Update: 2016-07-07

--- a/lib/irrlicht/CMakeLists.txt
+++ b/lib/irrlicht/CMakeLists.txt
@@ -2,11 +2,7 @@
 find_package(PNG REQUIRED)
 find_package(JPEG REQUIRED)
 
-include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include/"
-                    "${JPEG_INCLUDE_DIR}"
-                    "${PNG_INCLUDE_DIRS}"
-                    "${ZLIB_INCLUDE_DIR}"
-                    "${CMAKE_CURRENT_BINARY_DIR}/../zlib/") # For zconf.h on WIN32
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include/)
 
 find_package(OpenGL REQUIRED)
 include_directories(${OPENGL_INCLUDE_DIR})
@@ -508,7 +504,7 @@
 endif()
 
 add_library(stkirrlicht STATIC ${IRRLICHT_SOURCES})
-target_link_libraries(stkirrlicht ${PNG_LIBRARY} ${JPEG_LIBRARY} ${ZLIB_LIBRARY})
+target_link_libraries(stkirrlicht png jpeg z)
 
 if(WIN32)
     target_link_libraries(stkirrlicht imm32)
